packages/typography/src/Typography.js

```diff
@@ -55,6 +59,7 @@ export default class Typography extends Component {
       children,
       fontWeight,
       variant,
+      className,
```

Comment:
Instead of creating a new `prop` for `className`, could we just break it out from `otherProps` like `const { className } = otherProps and then pass it into the markup.

Reply:
You will probably have to change the order that `{ ...otherProps }` and `className` are passed in so the `className` from `...otherProps` doesn't override the `className` we want.

Reply:
Sure, understood your point and I have updated the code using your suggestions. Just checked the source code of `cx` function in `emotion`, it will automatically handle the args whatever its  variable type is, thus, I just pass the className prop to the `cx` function directly.

And shall I know why we should hide the className props? I saw in some other libraries like material-ui, element-ui, they expose classname, or classnames to users directly.

Reply:
Oh, another question: As in this way the className prop is not exposed directly, will we need remove "className" knob in storybook that i added yesterday?

Reply:
As the `className` prop will not be exposed directly now, i just removed my previous commit for adding className in storybook knobs.

Reply:
I feel like if we are using `className` for any type of additional functionality then we would want to expose that as a `prop`. But in this case, we are just passing it down from the user to the root element and not doing anything else with it so getting it from `otherProps` seems like the preferred way to go especially since `className/class` is a standard html attribute.